src/Ingredients/DisplayIngredients.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace BreadAndIfit\Ingredients;
+
+class DisplayIngredients
+{
+    /**
+     * This method calls the functions below and its purpose is to be called in the index.php to make
```

Comment:
too specific a comment. dont need to start with "This method"... we know its a method. also it doesnt only have to be used in index.php